src/cdk/layout/media-matcher.spec.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {LayoutModule, BreakpointObserver} from './index';
+import {MediaMatcher} from './media-matcher';
+import {async, TestBed, inject} from '@angular/core/testing';
+import {Platform} from '@angular/cdk/platform';
+
+describe('MediaMatcher', () => {
+  let breakpointManager: BreakpointObserver;
+  let mediaMatcher: MediaMatcher;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [LayoutModule]
+    });
+  }));
+
+  beforeEach(inject(
+    [BreakpointObserver, MediaMatcher],
+    (bm: BreakpointObserver, mm: MediaMatcher) => {
+      breakpointManager = bm;
+      mediaMatcher = mm;
+    }));
+
+  it('correctly returns a MediaQueryList to check for matches', () => {
+    expect(mediaMatcher.matchMedia('(min-width: 1px)').matches).toBeTruthy();
+    expect(mediaMatcher.matchMedia('(max-width: 1px)').matches).toBeFalsy();
+  });
+
+  it('adds css rules for provided queries when the platform is webkit.',
```

Comment:
Add a test for _not_ adding the styles on other platforms?

Reply:
I'm not sure how to go about this because it depends on what platform the test is being run on.

Reply:
It's fine for the test to have an `if` block checking the platform before doing anything